src/handleRequest.ts

```diff
@@ -83,12 +84,14 @@ export const handleRequestWith = (requestHandlers: RequestHandler[]) => {
 
       // Transform Headers into a list to be stringified preserving multiple
       // header keys. Stringified list is then parsed inside the ServiceWorker.
-      const responseWithHeaders = {
+      const responseWithHeaders: MockedResponse = {
         ...mockedResponse,
         // @ts-ignore
         headers: Array.from(mockedResponse.headers.entries()),
       }
 
+      log(req, responseWithHeaders, relevantRequestHandler)
```

Comment:
For debugging tests in the IDE, where grouping is not built in and all the text prints into the console, it would be very helpful to disable this logging.